packages/languages/pl/README.md

```diff
@@ -0,0 +1,46 @@
+# @zxcvbn-ts/language-pl
+
+The Polish dictionary and language package for zxcvbn-ts
+
+## Install
+
+### npm:
+
+`npm install @zxcvbn-ts/language-pl --save`
+
+### yarn:
+
+`yarn add @zxcvbn-ts/language-pl`
+
+## Setup
+
+```js
+import { zxcvbn, zxcvbnOptions } from '@zxcvbn-ts/core'
+import zxcvbnCommonPackage from '@zxcvbn-ts/language-common'
+import {default as pl} from '@zxcvbn-ts/language-pl'
+
+const password = 'somePassword'
+const options = {
+  translations: pl.translations,
+  graphs: zxcvbnCommonPackage.adjacencyGraphs,
+  dictionary: {
+    ...zxcvbnCommonPackage.dictionary,
+    ...pl.dictionary,
+  },
+}
+
+zxcvbnOptions.setOptions(options)
+
+zxcvbn(password)
+```
+
+## Sources
+
+1. First and last names data use to be a dictionary for zxcvbn-ts are extracted from files based on Polish Government Public Datasets (Public Domain) and available publicly on <https://github.com/oskar-gmerek/polish_database> (MIT licensed)
+  [*] Data is based on Last and First names registered under Polish Universal Electronic System for Population Register and may contains also names of foreigners registered under this Register, (ex. refugees from Ukraine who were given a PESEL number after the Russian Invasion of Ukraine)
```

Comment:
If you got the data from a public dataset. It would be nice if you use it directly and if necessary create an generator for the file like the other language packages if the data was available in the internet to download.
If this is not possible because its behind a login or whatever it would be nice if you write where you got the data from